applications/serial_lte_modem/src/slm_at_udp_proxy.c

```diff
@@ -487,11 +487,7 @@ static bool socket_is_in_use(void)
 	return true;
 }
 
-/**@brief handle AT#XUDPSVR commands
- *  AT#XUDPSVR=<op>[,<port>]
- *  AT#XUDPSVR? READ command not supported
- *  AT#XUDPSVR=?
- */
+/* Handles AT#XUDPSVR commands. */
 int handle_at_udp_server(enum at_cmd_type cmd_type)
```

Comment:
Why remove? It explains the parameters to be parsed.

Reply:
Same code style for all AT command handlers, not just UDP client

Reply:
Shorter description works for me. It is quite easy to see in the code which operations are supported.